src/components/DeviceSelector.js

```diff
@@ -9,30 +9,22 @@ import Menu from '@material-ui/core/Menu';
 
 const styles = theme => ({
   root: {
-    width: '100%',
-    maxWidth: 360
+    width: '100%'
+    // maxWidth: 360
   }
 });
-
-const options = [
-  'Show some love to Material-UI',
-  'Show all notification content',
-  'Hide sensitive notification content',
-  'Hide all notification content'
-];
-
 class DeviceSelector extends React.Component {
   state = {
-    anchorEl: null,
-    selectedIndex: 1
+    anchorEl: null
   };
 
-  handleClickListItem = event => {
-    this.setState({ anchorEl: event.currentTarget });
+  getActiveDevice = devices => {
+    if (devices.length < 0) return [];
+    return devices.filter(device => device.is_active)[0];
```

Comment:
What happens if the filter produces in an empty array?

Reply:
it breaks :(
let me fix that